src/ray/raylet/worker_pool.h

```diff
@@ -131,13 +131,6 @@ class WorkerPool {
   std::string WarningAboutSize();
 
  protected:
-  /// A map from the pids of starting worker processes
-  /// to the number of their unregistered workers.
-  std::unordered_map<pid_t, int> starting_worker_processes_;
-  /// The number of workers per process.
-  int num_workers_per_process_;
-
- private:
```

Comment:
We are we making all of the below `protected` instead of `private`? For the tests?

Reply:
I refactored the code and move some fields back to private. But I still need states_by_lang_ to be protected for tests.